Contents.m

```diff
@@ -0,0 +1,8 @@
+% MATLAB-bindings for coupling library preCICE
+% Version 2.1.0.1 (R2018b) 14-Oct-2020
```

Comment:
I do not really understand the `(R2018b)` in brackets. What does this mean?

If I call `ver` on my system I get

```
>> ver
-----------------------------------------------------------------------------------------------------
MATLAB Version: 9.6.0.1174912 (R2019a) Update 5
MATLAB License Number: XXXXXX
Operating System: XXXX
Java Version: XXXX
-----------------------------------------------------------------------------------------------------
MATLAB                                                Version 9.6         (R2019a)
MATLAB-bindings for coupling library preCICE          Version 2.1.0.1     (R2018b)
```

Reply:
The `(R2018b)` comes from the fact that I have `R2018b` installed on my system. I agree that this is misleading.
**Solution:** State all compatible MATLAB versions in the brackets OR state nothing. Stating nothing will look odd in the list.

Reply:
I am not sure whether we really can give reliable information about compatibility with MATLAB versions. For this our testing of the bindings is currently by far too basic. I think for now the best possibility (to avoid false information) is to write nothing. Do you know what other community toolboxes that do not ship with MATLAB do?

Reply:
The [README](https://github.com/precice/matlab-bindings/blob/develop/README.md) currently says that the bindings are tested for R2018b, R2019a and R2019b. We can mention these versions of MATLAB in the bracket or we can drop the whole thing. 
* I tried very briefly to look at what non-shipped toolboxes do but could not find a specific pattern.